locust/scenarios/locustfile.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+import subprocess
+import time
+
+from locust import HttpUser, task, between
+from urllib.parse import urlparse
+
+
+url = str(os.environ.get("HOST"))
+parsed_url = urlparse(url)
+host = parsed_url.hostname
+port = parsed_url.port
+
+logger_level = str(os.environ.get("LOGGER_LEVEL"))
+logger = logging.getLogger(__name__)
+logger.setLevel(logger_level)
+
+curve = str(os.environ.get("CURVE"))
```

Comment:
"Curve" is a pretty dated term from the time KEM could only do EC curves. Suggest renaming to "Group" which is the more standard term (also used in TLS1.3).

Reply:
Done [here](https://github.com/open-quantum-safe/oqs-demos/pull/304/commits/ab1d2f45c378d14f7b5c01a1e6a3b656a69b8b3c#diff-8a1eed52bbc906e4f897ac6f113321abdd9f6fa9c742de72ac721a6d0cb6f195R4)